src/SparseMFModel.cpp

```diff
@@ -134,7 +134,8 @@ void SparseMFModel::loadSerialized(const void* data, uint64_t minibatch_size, ui
   }
 
 #ifdef DEBUG
-  check();
+    // TODO: Uncomment when implemented.
+    // check();
```

Comment:
Why is this commented? This is very useful for debugging/testing.

Reply:
I had initially commented it because the check() function is unimplemented, so the print statements were not useful for me. The stdout is too verbose for Travis to run the MF test as is (it exceeds the maximum stdout size), so I've disabled the DEBUG flag in the meantime.